runtime-config.md

```diff
@@ -1,3 +1,14 @@
+# Runtime Configuration
+
+## Hooks
+
+Lifecycle of a container's runtime, for setup and teardown.
+Presently there are `Prestart` and `Poststop`.
+
+* `Prestart` is a list of hooks to be run before the container process is executed
+* `Poststop` is a list of hooks to be run after the container process exits
+	
+
```

Comment:
You removed a doubled blank line from `code-of-conduct.md` in this branch, so probably don't add one here ;).